lib/neo4j/core/cypher_session/adaptors/http.rb

```diff
@@ -16,7 +17,8 @@ def initialize(url, options = {})
           end
 
           def connect
-            @requestor = Requestor.new(@url, USER_AGENT_STRING, self.class.method(:instrument_request))
+            @requestor = Requestor.new(@url, USER_AGENT_STRING, self.class.method(:instrument_request),
+                                       @options[:faraday_options] || @options['faraday_options'] || {})
```

Comment:
I think that you should just be able to do `@options['faraday_options'] || {}`, right?